hardware/1Wire.cpp

```diff
@@ -188,10 +188,11 @@ void C1Wire::GetDeviceDetails()
 	if ((m_devices.size() == 0) || (m_sql.m_bAcceptNewHardware))

```

Comment:
No.
Before, devices always deleted and added.
Now, devices delete/add only when 
1) I do not have any (most likely the first run etc)
2) I allow to add new devices in settings
When in setting I disable "allow new devices" - this method call only one time (when program starting).
When "Allow new devices" is enabled - this working as before (always delete/add devices to list)